specs/protocol/README.md

```diff
@@ -81,6 +81,7 @@ The `metadata` object contains fields _about_ the resource and is present in _ev
 | `id`        | Y              | The resource's ID                           |
 | `createdAt` | Y              | ISO 8601 timestamp                          |
 | `updatedAt` | N              | ISO 8601 timestamp                          |
+| `protocolVersion` | Y        | Version of the protocol in use (x.x format) |
```

Comment:
do we foresee including other versions? if not, thoughts on using just `version`?

Reply:
Hmmmm naming is hard. Valid point's @kirahsapong! 

However, I think it would be unwise to venture into the realm of versioning messages & resources independently such that we have to consider compatibility across message versions & resource versions (imagine a world where an exchange is transpiring where an RFQ is of one version but then the Quote is of another version and somehow the software has to reconcile that.

So, I think it's safe to assume we'll version the protocol as a whole, which consists of all the message types and resources for that given version. I vote `version` is sufficient in conveying the intent here.

Reply:
oh yeah good point, interpretability ... meh makes sense to me but I'm not an outsider looking in so I have a bias

Reply:
> there prob still remains a world in which version support drops for one party halfway through an exchange,

This should not happen. We should note in the spec that version must remain the same across messages in a given exchange. I.e. if the `exchangeId` is the same, then `version` must be the same.

Reply:
@kirahsapong Not sure I understand. What's orthogonal to version support dropping halfway through an exchange? I'm attempting to define how to handle version support changing halfway through an exchange.

Reply:
I think i'm confused about what version support dropping halfway through an exchange actually means. Can you give an example of a scenario where this happens?


I imagine two parties would begin an exchange with a given version only if they support exchanges at that version of TBDex.

I can see a situation where Alice submits an RFQ to a PFI with a version that the PFI does not support. In that case, the PFI should return a 4XX response, not a Close message of a version that they don't support.

Reply:
+1 on `protocol`. 

For dropping support for a version, I suppose that's within the power of a PFI or a wallet to do mid-exchange. (And if we assume that there are a lot of exchanges happening simultaneously, it is almost unavoidable that a PFI/wallet may have exchanges for two different versions unless they do a hard cut over on version.)

We should think through this a bit more and go through the scenario of migrating from one version to another, to come up with a recommended strategy that can be added to the spec text.